proto/v1alpha1/rode_policy.proto

```diff
@@ -78,28 +78,44 @@ message ListPoliciesResponse {
 }
 
 message UpdatePolicyRequest {
-  // the auto-generate id of the occurrence
+  // the auto-generated id of the policy
```

Comment:
Thanks for adding comments to these fields.  Since these comments [end up in the generated go code](https://github.com/rode/rode/pull/87/files?diff=split&w=1#diff-7214255bb58bc2b68bb2e5f9cbc3ac333f6f071335357a83bb6c19c9d1e46e53R642), we should try to follow godoc convention for them.

```suggestion
  // Id is the auto-generated id of the policy
```